source/release_notes/5.0.9-8.rst

```diff
@@ -0,0 +1,37 @@
+.. _PSMDB-5.0.9-8:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.9-8 (2022-06-15)
+================================================================================
+
+:Date: June 15, 2022
+:Installation: :ref:`install`
+
+Percona Server for MongoDB 5.0.9-8 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.9 Community Edition.
+It supports MongoDB 5.0.9 protocols and drivers.
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* The support of encrypted KMIP client certificates and keys through the password setting. 
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* `SERVER-65636 <https://jira.mongodb.org/browse/SERVER-65636>`_ - Fixed the limits of LDAP connections per host from total number of connections to the number of in-progress connections.
```

Comment:
this is enterprise-only bug fix so it has nothing to do with PSMDB

Reply:
Removed